x-pack/auditbeat/module/system/login/login_test.go

```diff
@@ -0,0 +1,36 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package login
```

Comment:
This one also needs a build tag for linux only, otherwise tests fail on macOS.

Reply:
👍  - done